app/temp/mongo-example.js

```
@@ -0,0 +1,36 @@
+const mongoose = require('mongoose')
+mongoose.connect('mongodb://root:foobar@127.0.0.1:27017', {useNewUrlParser: true});
+
+const db = mongoose.connection;
+db.on('error', console.error.bind(console, 'Mongo Connection Error:'))
+db.once('open', function() {
+    console.log('Connected')
+
+    const kittySchema = new mongoose.Schema({
+        name: String,
+        breed: String
+    })
+
+    kittySchema.methods.speak = function() {
+        const greeting = this.name
+            ? `Meow name is ${this.name}, my breed is ${this.breed}`
+            : "I don't have a name!";
```

Outrageous! I would throw an error.